### PR TITLE
IDE-925 Investigate if ECLIDE/ClientTools requires Windows admin rights

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: comms/EclCC.cpp [hunk 1]
```diff
@@ -655,6 +655,11 @@ CEclCC * CreateEclCCRaw(IConfig * config, const std::_tstring & compilerFile)
     return EclCCCache.Get(new CEclCC(config, compilerFile));
 }
 
+boost::filesystem::path EclCCSubPath(boost::filesystem::path directory)
```
Comment (GordonSmith) [inline comms/EclCC.cpp]: Should be
```
EclCCSubPath(const boost::filesystem::path & directory)
```

**Comment by dehilsterlexis** [comment]
@GordonSmith back to you

File: comms/EclCC.cpp [hunk 2]
```diff
@@ -655,6 +655,11 @@ CEclCC * CreateEclCCRaw(IConfig * config, const std::_tstring & compilerFile)
     return EclCCCache.Get(new CEclCC(config, compilerFile));
 }
 
+boost::filesystem::path EclCCSubPath(boost::filesystem::path & directory)
```
Comment (GordonSmith) [inline comms/EclCC.cpp]: Should be:
```
boost::filesystem::path EclCCSubPath(const boost::filesystem::path & directory)
```
?

Comment (dehilsterlexis) [reply comms/EclCC.cpp]: @GordonSmith yes it should.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by GordonSmith** [comment]
@richardkchapman ready to pull

**Pull request merged**